Client/Frontend/Browser/TabLocationView.swift

```diff
@@ -229,6 +230,11 @@ class TabLocationView: UIView {
         } else {
             urlTextField.text = url?.absoluteString
         }
+        // remove https:// (the scheme) from the url when displaying
+        if let scheme = url?.scheme {
```

Comment:
This will remove more than just `https://`, it would also remove `http://`, right? If so, how will the user know they're on a secure site?

Reply:
The green lock that appears next to the URL.